Engine/Source/Runtime/ECWorld/LightComponent.h

```diff
@@ -145,7 +145,7 @@ class LightComponent final
 private:
 	U_Light m_lightUniformData;
 
-	bool m_isCastShadow;
+	float m_castShadowIntensity;
```

Comment:
Shadow intensity is controlled by lighting system's light intensity or sky component's sun light intensity. Please don't add a shadow intensity which will be strange to understand.